requirements.in

```diff
@@ -2,4 +2,4 @@ requests>=2.25.0,<3
 pydantic~=1.10.5
 graphene~=2.1.9
 graphene-sqlalchemy~=2.3.0
-git+https://github.com/Ensembl/ensembl-metadata-api.git@2.0.1a2#egg=ensembl-metadata-api
+git+https://github.com/Ensembl/ensembl-metadata-api.git@2.1.0.dev1#egg=ensembl-metadata-api
```

Comment:
installing dev version of metadata api

Reply:
Yeah, I am awaiting the api PR to be merge to do an actual version. But I need this to use the ReleaseStatus and DatasetStatus constants.

Reply:
Just updated to a more stable one (a1)